src/assets/Enums.ts

```diff
@@ -19,3 +19,31 @@ export enum STATUS {
     PROVISIONAL_OVER_CURRENT = "provisional_over_current",
     ALL = "all"
 }
+
+export enum PERMITTED_DANGEROUS_GOODS {
```

Comment:
These should probably be in the ITechRecord.d.ts since they're only enums for the purpose  of types, rather than how the enums in this file have normally been used.